js/bytex.js

```diff
@@ -670,60 +793,56 @@ module.exports = class bytex extends Exchange {
         ];
     }
 
+    parseBalance (response) {
+        const timestamp = this.parse8601 (this.safeString (response, 'updated_at'));
+        const result = {
+            'info': response,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+        };
+        const currencyIds = Object.keys (this.currencies_by_id);
+        for (let i = 0; i < currencyIds.length; i++) {
+            const currencyId = currencyIds[i];
+            const code = this.safeCurrencyCode (currencyId);
+            const account = this.account ();
+            account['free'] = this.safeString (response, currencyId + '_available');
+            account['total'] = this.safeString (response, currencyId + '_balance');
+            result[code] = account;
+        }
+        return this.safeBalance (result);
+    }
+
     async fetchBalance (params = {}) {
+        /**
+         * @method
+         * @name bytex#fetchBalance
+         * @description query for balance and get the amount of funds available for trading or funds locked in orders
+         * @param {dict} params extra parameters specific to the bytex api endpoint
+         * @returns {dict} a [balance structure]{@link https://docs.ccxt.com/en/latest/manual.html?#balance-structure}
+         */
         await this.loadMarkets ();
         const response = await this.privateGetUserBalance (params);
         //
         //     {
-        //         "user_id": '...',
+        //         "updated_at": "2020-03-02T22:27:38.428Z",
```

Comment:
I don't get the key `updated_at` when I call this method, I get `user_id`

You can call it from the command line like `node examples/js/cli.js bytex --verbose` to get the raw exchange response. (You'll probably need to run `npm run export-exchanges` prior)